pallets/ethereum-checked/src/mock.rs

```diff
@@ -294,7 +294,7 @@ impl ExtBuilder {
             assert_ok!(Evm::create2(
                 RuntimeOrigin::root(),
                 ALICE_H160,
-                hex::decode(STORAGE_CONTRACT).unwrap(),
+                hex::decode(STORAGE_CONTRACT).expect("invalid code hex"),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code is currently using `unwrap()` to decode the `STORAGE_CONTRACT` hex value. It's recommended to use `expect()` instead and provide a meaningful error message in case the decoding fails. This will help with debugging in case of any issues.

```diff
-                hex::decode(STORAGE_CONTRACT).unwrap(),
+                hex::decode(STORAGE_CONTRACT).expect("Failed to decode STORAGE_CONTRACT hex"),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->